### PR TITLE
[Draft] Provide welcome guide via ONE view container

### PR DESCRIPTION
**Pull request by yunjayh**
[WIP]

This commit provides welcome guide via ONE view container.

ONE-vscode-DCO-1.0-Signed-off-by: yunjay-hong <yunjay.hong@samsung.com>

---

for #932

### CODE REVIEW

**Comment by yunjayh** [comment]
**[The texts are work in progress]** IMHO with this way, user can get what to do after installing our extension, or opening new window without workspace opened.
image

File: package.json [hunk 1]
```diff
@@ -16,6 +16,16 @@
   ],
   "main": "./out/extension.js",
   "contributes": {
+		"viewsWelcome": [
+      {
+        "view": "OneExplorerView",
+        "contents": "Open folder where model files are stored. Models files are one of the following format:\n- *.tflite, *.pb, Keras model or saved model for Tensorflow​\n- *.onnx or *.pth for PyTorch\n[Open Folder](Command:onevscode.fileTreeView-open)\n"
```
Comment (dayo09) [inline package.json:22]:
```suggestion
        "contents": "Open folder where model files are stored.\nModel files are one of the following format:\n- *.tflite, *.pb, Keras model or saved model for Tensorflow​\n- *.onnx or *.pth for PyTorch\n[Open Folder](Command:onevscode.fileTreeView-open)\n"
```

Comment (yunjayh) [reply package.json:22]: Does `Keras model` or `saved model` and `.pth` are visible in one-explorer? I read the codes, but I'm not sure about it.
/cc @dayo09

**Comment by yunjayh** [comment]
image

File: src/OneExplorer/OneExplorer.ts [hunk 2]
```diff
@@ -501,10 +501,21 @@ input_path=${modelName}.${extName}
...
 
   constructor(context: vscode.ExtensionContext) {
+    try {
+      this.workspaceRoot = vscode.Uri.file(obtainWorkspaceRoot());
+    } catch (e: unknown) {
+      let errmsg = 'Failed to obtain workspace root';
+      if (e instanceof Error) {
+        errmsg = e.message;
+      }
+      Logger.debug('OneExplorer', errmsg);
```
[the remark this answers is not in the document]
Comment (yunjayh) [reply src/OneExplorer/OneExplorer.ts:516]: Yes I intended it. I wrote that code when I was trying to test how to show welcome view in one-explorer view. Now, I'm working on how to treat *no workspace error* in proper way in my local draft. Anyway, thank you for your advice!!

**Comment by yunjayh** [comment]
@dayo09 Could you please take a look at `OneExplorer.ts`?
I modified it not to throw error and not to pop-up the error message balloon. Is there any threat that I didn't consider or expect?

AFAIK, the extension's `activation` function will be executed in every vscode window start up ([source](https://code.visualstudio.com/api/references/activation-events#onStartupFinished)). 

If vscode cannot obtain the workspace, `OneTreeDataProvider` returns empty children. It leads to show welcome view which defined at `package.json`.
Else, if workspace is defined by user, `OneExplorerView` works well as expected.

**Pull request closed without merging**